### DISCUSSION TITLE
IPV6 Support

### DISCUSSION THREAD

**Discussion by STR07** [requarks/wiki#6296] [Help / Questions] [answered]
When will Wikijs get IPv6 support? 
Thanks in advance

#5701

**Comment by STR07** [comment]
Or is there maybe a config setting so I can turn on ipv6 support ? @NGPixel 
Thanks

WikiJS: 2.5.298
OS: Debian 11
DB: PostgreSQL 14.7
WebServer: Node.js 16.19.1

Or are you planning to Support IPV6 in Version 3.0?  @NGPixel

**Reply by ciphermenial** [reply]
IPv6 is supported. However nodemailer sends emails over IPv4 and not IPv6. To listen on IPv6 you set the bindIP to [::]

**Reply by STR07** [reply]
```yml
# ---------------------------------------------------------------------
# IP address the server should listen to
# ---------------------------------------------------------------------
# Leave 0.0.0.0 for all interfaces

#bindIP: 0.0.0.0

 bindIP: [::]
```

Unfortunately, the BIND: [::] does not work. Unfortunately, the server cannot even be reached via ipv4. Did I perhaps misunderstand your solution? 
Thanks

**Reply by ciphermenial** [reply]
Nope, you haven't misunderstood. That is exactly how I have mine configured and it works perfectly.

**Reply by NGPixel** [reply, marked as answer]
Try `bindIP: '::'` instead.

**Reply by STR07** [reply]
It works now. I can reach the website via ipv4 and ipv6(dual-Stack). 
Thank you very much for your help :)
